shared/common-adapters/kb-text-input.native.tsx

```diff
@@ -44,31 +47,48 @@ if (isAndroid) {
 
       const textContainer = (
         <NativeTextInput
+          // @ts-ignore no RN types
```

Comment:
Are the `@types/react-native` any good?

Reply:
Tried locally a bit and they seem good, we run into this issue: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/26218. I wonder if there's a way to work around without having to run `tsc` twice.